stories/patterns/ux-patterns/feedback-yes-no/examples/docs/feedback-yes-no-example.tsx

```diff
@@ -107,19 +127,28 @@ class Feedback extends React.PureComponent<{
 }
 
 class FeedbackYesNo extends React.PureComponent {
-  state = { status: 'default', visible: true, feedbackType: null };
+  state = {
+    status: 'default',
+    visible: true,
+    feedbackType: null,
+    value: { feedback: '', email: '' },
+  };
   timeout: any;
   onSubmit = () => {
     this.requestServer('success', 1000);
     this.setState({ status: 'loading' });
   };
-  requestServer = (status, time, cb?: () => void) => {
+  onChange = (e: any, trigger: string) => {
```

Comment:
`React.SyntheticEvent` ?

And you could get `id` from that event, so, you don't need a `trigger`, I think